bb/deps.edn

```diff
@@ -1,8 +1,8 @@
 {:paths ["src" "resources"]
  :deps {org.babashka/spec.alpha {:git/url "https://github.com/babashka/spec.alpha"
-                                 :git/sha "644a7fc216e43d5da87b07471b0f87d874107d1a"}
+                                 :git/sha "951b49b8c173244e66443b8188e3ff928a0a71e7"}
         ;; https://github.com/babashka/tools.bbuild
-        io.github.babashka/tools.bbuild {:git/sha "0a4959d2b1147f79e9fb37e0243727390b8ba2b3"}
+        io.github.babashka/tools.bbuild {:git/sha "a3c0723b6250756eaeed532ade5d6474df24e80a"}
         ;; https://github.com/borkdude/gh-release-artifact
-        io.github.borkdude/gh-release-artifact {:git/sha "05f8d8659e6805d513c59447ff41dc8497878462"}
+        io.github.borkdude/gh-release-artifact {:git/sha "db5e79559fdbfa009ed4a0921079e20e1e76b269"}
```

Comment:
I cannot decide if it would be too dangerous to upgrade the build packages too often or if it is not too much of a problem if a build or release fails from time to time.

Reply:
Ok, I already had to exclude the tools.bbuild because the most recent version made the builds fail.